### PR TITLE
Clean up package

### PR DESCRIPTION
**Pull request by lazappi**
Errors in parsing responses were due to the package name not being set when the package was generated. I have regenerated the package using this command:

```shell
java -Dcolor -jar openapi-generator-cli.jar generate
  -g r
  --input-spec https://us-west-2.api.lamin.ai/openapi.json
  --output laminr.api
  --package-name laminr.api
  --additional-properties=packageName=laminr.api,exceptionPackage=rlang
```

This also uses **{rlang}** for exceptions which should help with clearer error messages in the future.

### CODE REVIEW

**Comment by lazappi** [comment]
@rcannood This needs some tidying to clean up things we don't want to change but it fixes the errors I was having using the package

**Comment by rcannood** [comment]
Oh nice, thanks for this!

I suppose that means that there are more settings we can change further get the package to do what we want.

**Comment by lazappi** [comment]
@rcannood I think this would be ok to use for testing now but there are still some things that need to be tidied up (checks, versions, license etc.). Not sure if you want to review/merge now or do all that first.

File: update_package.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+echo "Generating R client package using OpenAI Generator..."
+java -Dcolor -jar $GeneratorJar generate \
+    -g r \
+    --input-spec https://us-west-2.api.lamin.ai/openapi.json \
+    --output . \
+    --package-name laminr.api \
+    --additional-properties=packageName=laminr.api,exceptionPackage=rlang \
+
+echo "Documenting package..."
+R -e "devtools::document()"
```
Comment (rcannood) [inline update_package.sh:19]:
```suggestion
Rscript -e "devtools::document()"
```

File: update_package.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    exit 1
+fi
+
+echo "Generating R client package using OpenAI Generator..."
+java -Dcolor -jar $GeneratorJar generate \
+    -g r \
+    --input-spec https://us-west-2.api.lamin.ai/openapi.json \
+    --output . \
+    --package-name laminr.api \
+    --additional-properties=packageName=laminr.api,exceptionPackage=rlang \
```
Comment (rcannood) [inline update_package.sh:16]:
```suggestion
java -Dcolor -jar $GeneratorJar generate \
    -g r \
    --input-spec $spec \
    --output . \
    --package-name $pkgname \
    --additional-properties=packageName=$pkgname,exceptionPackage=rlang,packageVersion=$pkgver

File: update_package.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+GeneratorJar=${OPENAI_GENERATOR_PATH:-}
+
+if [ -z "${GeneratorJar}" ]; then
+    echo "OPENAI_GENERATOR_PATH is not set. Please set it to the path of the openapi-generator-cli.jar file."
+    exit 1
+fi
+
```
Comment (rcannood) [inline update_package.sh:9]:
```suggestion

spec=https://us-west-2.api.lamin.ai/openapi.json
pkgname=laminr.api
pkgver=$(curl $spec | jq -r .info.version)

```

**Review by rcannood — COMMENTED** [review]
minor suggestions

**Comment by lazappi** [comment]
@rcannood The remaining check warning is because of the license. Happy to change if you tell me what to. The NOTEs aren't really avoidable without manually editing things.

**Comment by rcannood** [comment]
Let's go with Apache because Lamin seems to use this as a license consistently throughout their repos

**Review by rcannood — APPROVED** [review]

**Pull request merged**